napari/_vispy/vispy_image_layer.py

```diff
@@ -36,7 +36,10 @@ def __init__(self, layer):
         )
         self.layer.events.gamma.connect(lambda e: self._on_gamma_change())
         self.layer.events.iso_threshold.connect(
-            lambda e: self._on_iso_threshold_change()
+            lambda e: self._on_threshold_change()
```

Comment:
along the lines of the previous comments, but slightly different... not sure what your personal preference here is (and I know this is older code), but if there's a method like this that is more or less designed to be a vispy event handler, even if it doesn't use the event object, I think it'd be nice if we added `event=None` to the method so that we don't have to catch the event and discard it with a lambda.  so this would be:
```python
self.layer.events.iso_threshold.connect(self._on_threshold_change)
...
def _on_threshold_change(self, event=None):
...
```
(same for `self.layer.events.attenuation.connect` below)

Reply:
👍 on this change, which I campaigned for originally. =)

Reply:
@sofroniewn, curious to hear what you think.  if you're for it, another option is to ignore the change here, and I can follow up with a PR to alter that pattern for uniformity across the repo.  On quick check, there are roughly 35 places where the `events.__.connect(lambda e: self._do_something())` pattern is used.  And another 25 or so more qt signal connections that could be simplified in one way or another

Reply:
That's a lot! I'm absolutely fine with the changes being made, I remember @jni mentioning this and I don't have objections, but given the magnitude of the change I think a follow-up dedicated PR is warranted

Reply:
cool, may have a couple questions here and there, but will mark them inline if I do. feel free to ignore here